packages/libs/src/services/discoveryProvider/DiscoveryProvider.ts

```diff
@@ -95,6 +97,17 @@ type DiscoveryNodeChallenge = {
   disbursed_amount: number
 }
 
+const getUserIdOverride = async (localStorage?: LocalStorage) => {
+  try {
+    const userIdOverride = await localStorage?.getItem(
+      '@audius/user-id-override'
+    )
+    return userIdOverride == null ? undefined : userIdOverride
+  } catch {
+    return undefined
```

Comment:
might be useful to have a log here saying there was an issue

Reply:
Good call 👍